helix-term/src/commands.rs

```diff
@@ -1162,10 +1162,6 @@ fn goto_file_impl(cx: &mut Context, action: Action) {
     let (view, doc) = current_ref!(cx.editor);
     let text = doc.text();
     let selections = doc.selection(view.id);
-    let rel_path = doc
```

Comment:
I believe this changes the behavior of `gf` a little: the path should be interpreted as relative to the current document. Also see https://github.com/helix-editor/helix/pull/8126 which aims to change that behavior

Reply:
Okay I can update it. Just to clarify, are we going with the original wording of that issue (not sure if it even applies still) or the clarification here:

https://github.com/helix-editor/helix/pull/8126#issuecomment-1712944308

Just resolve relative to the current file path no matter what?

Reply:
I would prefer the behavior from Pascal's comment. I think this makes sense:

1. Check if the file exists relative to the current doc
2. Check if the file exists relative to the CWD
3. Otherwise open a new file relative to the current doc